packages/gestalt/src/Layer.js

```diff
@@ -32,6 +31,13 @@ export default function Layer({
         document.body.removeChild(element.current);
       }
     };
+  }, []);
+
+  useEffect(() => {
+    if (element.current) {
+      element.current.style.zIndex = zIndex === null ? '' : `${zIndex}`;
+      element.current.className = zIndex === null ? '' : styles.layer;
+    }
```

Comment:
```suggestion
    if (!element.current) {
      return;
    }
    element.current.style.zIndex = zIndex === null ? '' : `${zIndex}`;
    element.current.className = zIndex === null ? '' : styles.layer;
```

Reply:
done and also changed to `null` check to `undefined` check.